test/retest/repository_test.rb

```diff
@@ -160,7 +160,7 @@ def test_no_matches_with_multiple_possible_tests
       )
 
       @subject.cache = mock_cache
-      @subject.prompt = Prompt.new(input: StringIO.new("5\n"), output: StringIO.new)
+      @subject.prompt = Prompt.new(input: StringIO.new("6\n"), output: StringIO.new)
```

Comment:
We should probably stick to the 'none' value being '0' here.
It's a fortunate scenario where an out of range input will default to `nil`. This is the correct behaviour.

Reply:
Right, this should be `none` 👍